homeassistant/components/braviatv/config_flow.py

```diff
@@ -68,11 +75,13 @@ async def init_device(self, pin):
 
     @staticmethod
     @callback
-    def async_get_options_flow(config_entry):
+    def async_get_options_flow(config_entry: ConfigEntry) -> BraviaTVOptionsFlowHandler:
         """Bravia TV options callback."""
         return BraviaTVOptionsFlowHandler(config_entry)
 
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
         """Handle the initial step."""
         errors = {}
```

Comment:
Annotate empty collections.
```suggestion
        errors: dict[str, str] = {}
```